### DISCUSSION TITLE
In View Content fields are not showing for table with lot of fields

### DISCUSSION THREAD

**Discussion by junedc** [Sequel-Ace/Sequel-Ace#1770] [Q&A] [answered]
I have a table with 15 or more fields and when selecting that table and then press Content button I can only see the first few fields.  I have moved my mouse around the panel/screen but cannot find any horizontal bar that will enable me to see the rest of the fields/data.     Is there any short cut or settings that I need to enable to show the rest of the field/data?   Thanks.

**Comment by Jason-Morcos** [comment]
I think you just need to scroll horizontally

**Reply by junedc** [reply]
i have moved my mouse to every possible corner of the Sequel Ace panel, sidebar and anywhere I can think off the horizontal bar doesn't show.   Cheers.

**Comment by samuelbradshaw** [comment, marked as answer]
Here's a related issue that has a couple of potential workarounds/solutions: https://github.com/Sequel-Ace/Sequel-Ace/issues/1228

**Reply by junedc** [reply]
Using a Shift + Wheel mouse button works.   Thanks @samuelbradshaw for pointing that out.
